GCDWebServer/Core/GCDWebServerRequest.h

```diff
@@ -158,6 +158,30 @@ extern NSString* const GCDWebServerRequestAttribute_RegexCaptures;
 @property(nonatomic, readonly) BOOL acceptsGzipContentEncoding;
 
 /**
```

Comment:
You copy pasted without updating the doc: "of the connection" -> "for the request"